src/smb.ts

```diff
@@ -18,11 +18,12 @@ interface BaseAllocationRequest {
 }
 
 export class SmbProtocol {
-  async allocateConference(smbUrl: string): Promise<string> {
+  async allocateConference(smbUrl: string, smbKey: string): Promise<string> {
     const allocateResponse = await fetch(smbUrl, {
       method: 'POST',
       headers: {
-        'Content-Type': 'application/json'
+        'Content-Type': 'application/json',
+        Authorization: smbKey !== '' ? `Bearer ${smbKey}` : ''
```

Comment:
instead of including an empty `Authorization` header, how about using this syntax:

```typescript
...(smbKey && { Authorization:  `Bearer ${smbKey}` })
```

Reply:
I want it to be explicitly passed as an empty string and not an undefined variable as I wanted type checks to find out where I needed to adapt the call and not to accidentally miss one.

Reply:
But you are right, it is probably better to skip the header if not set.